schemas/expconf/v0/entrypoint.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://determined.ai/schemas/expconf/v0/entrypoint.json",
```

Comment:
out of curiosity, why go the Entrypoint-as-an-object route, vs the way you had it before, with the `"items": {"type": "string"}`?

Actually... is entrypoint.json actually used anywhere?

Reply:
my bad, i had originally had this "Entrypoint-as-an-object route" implementation, but in the end went with the "items" as before. so this object was not being used. i've deleted it.